src/components/SearchBar/index.tsx

```diff
@@ -37,6 +40,7 @@ export default function SearchBar({
   inputProps,
   classNames,
   styles,
+  renderItem,
```

Comment:
could we add a default initialization to `(el) => el` so the previous behavior is preserved by default

Reply:
We don't need to. This function is optional, if not present the behavior is the normal one rendering the text.